src/services/inplayer.account.service.ts

```diff
@@ -44,32 +44,62 @@ export const logout = async () => {
 export const getUser = async (): Promise<Customer> => {
   try {
     const { data } = await InPlayer.Account.getAccountInfo();
-    return processInplayerAccount(data);
+    return processAccount(data);
   } catch {
     throw new Error('Failed to fetch user data.');
   }
 };
 
 export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
+export const updateCustomer: UpdateCustomer = async (values) => {
+  try {
+    const fullName = `${values.firstName} ${values.lastName}`;
+    const response = await InPlayer.Account.updateAccount({
+      fullName,
+      metadata: {
+        ...(values?.consents && { consents: JSON.stringify(values.consents) }),
+        first_name: values.firstName?.replace(/\s\s+/g, ' ')?.trim() || '',
```

Comment:
Why did you end up replacing multi-spaces with single spaces if we aren't using a double space between first and last name?

Reply:
should we allow more than one space in the name? 
e.g. `John     Doe  `